third_party/terraform/resources/resource_compute_router_nat.go

```diff
@@ -78,80 +28,109 @@ func resourceComputeRouterNat() *schema.Resource {
 		},
 
 		Schema: map[string]*schema.Schema{
-			"name": &schema.Schema{
+			"name": {
 				Type:         schema.TypeString,
 				Required:     true,
 				ForceNew:     true,
 				ValidateFunc: validateRFC1035Name(2, 63),
 			},
-			"router": &schema.Schema{
+			"router": {
 				Type:     schema.TypeString,
 				Required: true,
 				ForceNew: true,
 			},
-			"nat_ip_allocate_option": &schema.Schema{
+			"nat_ip_allocate_option": {
 				Type:         schema.TypeString,
 				Required:     true,
-				ForceNew:     true,
 				ValidateFunc: validation.StringInSlice([]string{"MANUAL_ONLY", "AUTO_ONLY"}, false),
 			},
-			"nat_ips": &schema.Schema{
+			"nat_ips": {
 				Type:     schema.TypeSet,
 				Optional: true,
-				ForceNew: true,
 				Elem:     &schema.Schema{Type: schema.TypeString},
 			},
-			"source_subnetwork_ip_ranges_to_nat": &schema.Schema{
+			"source_subnetwork_ip_ranges_to_nat": {
 				Type:         schema.TypeString,
-				Optional:     true,
-				ForceNew:     true,
+				Required:     true,
```

Comment:
What happened if a user didn't set this before?

Reply:
they would get an error from the API if it's not set, and we don't set a default - i think it's ok to treat it less like a breaking change and more like a bugfix?

Reply:
Yep! Thanks.